app/controllers/v0/decision_review_evidences_controller.rb

```diff
@@ -22,7 +22,8 @@ def save_attachment_to_cloud!
         downstream_system: 'AWS S3',
         params: {
           # `form_attachment` is declared in `FormAttachmentCreate`, included above.
-          form_attachment_guid: form_attachment&.guid
+          form_attachment_guid: form_attachment&.guid,
+          encrypted: !filtered_params[:password].nil?
```

Comment:
i wonder if we might be able to flip this and do `filtered_params[:password].present?` to help with readability?

Reply:
Sure, will do 👍